modules/product/src/ProductAttributeValueStorageInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\commerce_product;
+
+use Drupal\Core\Entity\ContentEntityStorageInterface;
+
+/**
+ * Defines an interface for product attribute value entity storage classes.
+ */
+interface ProductAttributeValueStorageInterface extends ContentEntityStorageInterface {
+
+  /**
+   * Load entities by their property values and sort by their weights.
```

Comment:
Loads product attribute values for the given product attribute.

Then in @return indicate that the results are sorted by weight.